metrics/core/src/main/java/org/hyperledger/besu/metrics/prometheus/PrometheusMetricsSystem.java

```diff
@@ -141,6 +143,27 @@ public void createGauge(
     }
   }
 
+  @Override
+  public LabelledGauge createLabelledGauge(
+      final MetricCategory category,
+      final String name,
+      final String help,
+      final String... labelNames) {
+    final String metricName = convertToPrometheusName(category, name);
+    if (isCategoryEnabled(category)) {
+      final Consumer<Collector> addCollectorLambda =
+          (collector ->
+              collectors
+                  .computeIfAbsent(
+                      category, key -> Collections.newSetFromMap(new ConcurrentHashMap<>()))
+                  .add(collector));
+      final PrometheusGauge gauge =
+          new PrometheusGauge(metricName, help, List.of(labelNames), addCollectorLambda);
```

Comment:
Shouldn't this just be calling `addCollectorUnchecked`?
```suggestion
      final PrometheusGauge gauge =
          new PrometheusGauge(metricName, help, List.of(labelNames), collector -> addCollectorUnchecked(category, collector));
```